sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -198,47 +233,104 @@ public partial class RegistryArtifact
+        public virtual Azure.Response<Azure.Containers.ContainerRegistry.ArtifactUploadInfo> Upload(Azure.Containers.ContainerRegistry.ArtifactStreams content, Azure.Containers.ContainerRegistry.ArtifactUploadOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Containers.ContainerRegistry.ArtifactUploadInfo> Upload(string path, Azure.Containers.ContainerRegistry.ArtifactUploadOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Containers.ContainerRegistry.ArtifactUploadInfo>> UploadAsync(Azure.Containers.ContainerRegistry.ArtifactStreams content, Azure.Containers.ContainerRegistry.ArtifactUploadOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Containers.ContainerRegistry.ArtifactUploadInfo>> UploadAsync(string path, Azure.Containers.ContainerRegistry.ArtifactUploadOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
 namespace Azure.Containers.ContainerRegistry.Specialized
 {
+    public abstract partial class ArtifactManifest
+    {
+        protected ArtifactManifest() { }
+        public int? SchemaVersion { get { throw null; } set { } }
+    }
     public partial class ContainerRegistryBlobClient
     {
         protected ContainerRegistryBlobClient() { }
+        public ContainerRegistryBlobClient(string url) { }
+        public ContainerRegistryBlobClient(string url, Azure.Containers.ContainerRegistry.ContainerRegistryClientOptions options) { }
         public ContainerRegistryBlobClient(System.Uri endpoint, Azure.Core.TokenCredential credential, string repository) { }
         public ContainerRegistryBlobClient(System.Uri endpoint, Azure.Core.TokenCredential credential, string repository, Azure.Containers.ContainerRegistry.ContainerRegistryClientOptions options) { }
         public ContainerRegistryBlobClient(System.Uri endpoint, string repository) { }
         public ContainerRegistryBlobClient(System.Uri endpoint, string repository, Azure.Containers.ContainerRegistry.ContainerRegistryClientOptions options) { }
         public virtual System.Uri Endpoint { get { throw null; } }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
         public virtual string RepositoryName { get { throw null; } }
+        public virtual Azure.Response CancelUpload(string nextLink, Azure.RequestContext context = null) { throw null; }
```

Comment:
- May want to rename `nextLink` parameter, or hide it since we already know the link?
- Is there any reason for a convenience method?